client/src/components/questions.tsx

```diff
@@ -65,7 +65,15 @@ function Questions(props: {
   });
 
   const historyComponent = shouldDisplayPortrait() ? (
-    <HistoryChat height={500} />
+    <Chat
+      height={500}
+      chatProps={{
+        displayMentorNames: true,
+        height: 500,
+        width: "",
```

Comment:
can `width` and `bubbleColor` be optional props (just don't specify them when you don't want to override the default)

Reply:
done